Ukoly/DU_4/Caesar.py

```diff
@@ -0,0 +1,52 @@
+# vytvoření seznamu čísel, která jsou kódy pro písmena abecedy
+rank = list(range(65, 123))
+czech = ['ě', 'š', 'č', 'ř', 'ž', 'ý', 'á', 'í', 'é', 'ň', 'ď', 'ů', 'ú']
+for i in range(91, 97):
+    rank.remove(i)
+
+key = (input('Key: '))
+while key.isnumeric() is False or (int(key) < 0):
+    print('Key has to be positive number.')
+    key = (input('Key: '))
+key = int(key) % 26
+
+plaintext = input('Plaintext: ')
+
+# vytvoření seznamu, kam se budou přidávat zakázané znaky
+forbiden = list()
+for letter in plaintext:
+    if letter.lower() in czech:
+        forbiden.append(letter)
+
+# ošetření zadání stringu s českými znaky:
+while len(forbiden) > 0:
```

Comment:
Toto práve nefunguje tak ako by si chcela... ak to tam nacpeš znova aj s "zakázanými" znakmi, tak to prejde... :disappointed: